src/trace/trigger.ts

```diff
@@ -248,7 +250,11 @@ export function extractTriggerTags(event: any, context: Context) {
   }
```

Comment:
Above this is "parseEventSource". Just want to double check nothing in there can throw either. Other than that looks good.

Reply:
Yup I double checked that as well to see if anything in `parseEventSource` might call a non-existent property too. Should be all good.